commands/src/main/java/org/wildfly/extras/creaper/commands/auditlog/AbstractAuditLogSyslogReliableProtocolCommand.java

```diff
@@ -0,0 +1,37 @@
+package org.wildfly.extras.creaper.commands.auditlog;
+
+abstract class AbstractAuditLogSyslogReliableProtocolCommand extends AbstractAuditLogSyslogHandlerCommand {
```

Comment:
OK, looking at the class now, I think it doesn't make a lot of sense, it's there just for 2 common fields. If you really feel like it should stay, then keep it in place, but I'd rather inline those 2 fields into those 2 subclasses to get rid of this level of inheritance.